plugins-extra/query_enhancements/server/routes/query_assist/agents.ts

```diff
@@ -0,0 +1,62 @@
+import { ApiResponse } from '@opensearch-project/opensearch';
+import { RequestBody, TransportRequestPromise } from '@opensearch-project/opensearch/lib/Transport';
+import { RequestHandlerContext } from 'src/core/server';
+
+const ML_COMMONS_API_PREFIX = '/_plugins/_ml';
+const AGENT_REQUEST_OPTIONS = {
```

Comment:
this should be an advanced setting and pull from that: https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/plugins/data/server/ui_settings.ts

i will work on adding the base one and people can add to it